getting-started/01-overview.story.mdx

```diff
@@ -0,0 +1,73 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import './css/global.css';
+import logo from './assets/rainbow-logo.svg';
+import puzzle from './assets/puzzle.svg';
+import accesibility from './assets/accesibility.svg';
+import translation from './assets/translation.svg';
+
+<Meta title="Introduction|Getting Started/Overview" />
+
+<div>
+    <h1 class="page-title">
+        <img src={logo} width="65px" />
+        <span>rainbow-modules</span>
+    </h1>
+</div>
+
+Rainbow Modules includes a collection of components, fully integrated with [Firebase](https://firebase.google.com/), that will help you to build apps. Give it a hack and let us know what you think.
+
+<div align="center">
+
+[![React](https://badgen.net/badge/React/16.8.3/blue)](#)
+[![npm](https://badgen.net/badge/npm/8.31.0/blue)](#)
+[![dependencies](https://badgen.net/badge/dependencies/up%20to%20date/yellow)](#)
+[![devDependencies](https://badgen.net/badge/dev%20dependencies/up%20to%20date/green)](#)
+
+</div>
+
+<div class="flex-container flex-row flex-wrap">
+<div class="feature-list_item">
+<div class="feature-image">
+    <img src={puzzle} />
+</div>
+
+#### 60+ Components
+
+More than 60 components built on top of React.
+
+</div>
+<div class="feature-list_item">
+<div class="feature-image">
+    <img src={accesibility} />
+</div>
+
+#### Accessibility
+
+Accessibility is part of our definition of done. It's a must have for each component.
+
+</div>
+<div class="feature-list_item">
+<div class="feature-image">
+    <img src={translation} />
+</div>
+
+#### Internacionalization
+
+We designed each component with i18n in mind. Any new component or feature we may.
+
+</div>
+</div>
+
+### Installing from npm
+
+---
+
+Rainbow Modules is available as an [npm package](https://www.npmjs.com/package/rainbow-modules).
```

Comment:
@TahimiLeonBravo also check here since this URL does not work, maybe we can use something like this:
`https://www.npmjs.com/search?q=@rainbow-modules`